framework/src/outputs/Exodus.C

```diff
@@ -260,8 +260,10 @@ Exodus::outputNodalVariables()
     _exodus_io_ptr->write_timestep_discontinuous(
         filename(), *_es_ptr, _exodus_num, time() + _app.getGlobalTimeOffset());
   else
+  {
     _exodus_io_ptr->write_timestep(
         filename(), *_es_ptr, _exodus_num, time() + _app.getGlobalTimeOffset());
+  }
```

Comment:
new unecessary braces huh? That's random!

Reply:
Oops, I had a print statement in there for debugging.